packages/admin-sdk/src/ui/modal/index.ts

```diff
@@ -24,7 +24,7 @@ export type uiModalClose =
 export type buttonProps = {
   method: () => void,
   label: string,
-  variant?: 'primary'|'ghost'|'danger'|'ghost-danger'|'contrast'|'context',
-  size?: 'x-small'|'small'|'large',
+  variant?: 'primary'|'secondary'|'critical'|'action',
+  size?: 'x-small'|'small'|'default'|'large',
```

Comment:
This is a breaking change. But right now it's already broken anyway. @shopware/product-cc-framework thoughts on this?

To prevent further miss-matches, we could export the Button Props from Meteor and import them here.